src/models/health.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const db = require('../lib/db')
+
+function * getDbHealth () {
+  const isConnected = yield db.isConnected()
+  return isConnected ? {
+    status: 'OK'
+  } : {
+    status: 'KO'
```

Comment:
@clark800 do you have suggestions for something more standard? Otherwise I'll use the convention in the connector.

@gip do you think we need more than the db status? Unlike the connector, the ledger doesn't have multiple backends.